lib/slither/generator.rb

```diff
@@ -10,8 +10,8 @@ def generate(data)
 	    @definition.sections.each do |section|
 	      content = data[section.name]
 	      if content
-  	      content = [content] unless content.is_a?(Array)
-  	      raise(Slither::RequiredSectionEmptyError, "Required section '#{section.name}' was empty.") if content.empty?
+  	      content = [content] if content.is_a?(Hash)
```

Comment:
I notice the original spec omitted the test case for "when content is not Array". Since you are updating this to check "when content is Hash", let's add in the test case.

Reply:
It seems the indentation of the `if-else` block is off? 👀